openmetadata-ui/src/main/resources/ui/src/pages/MyDataPage/MyDataPage.component.tsx

```diff
@@ -189,17 +189,21 @@ const MyDataPage = () => {
             return pre.map((thread) => {
               const posts = thread.posts.filter((post) => post.id !== id);
 
-              return { ...thread, posts: posts };
+              return {
+                ...thread,
+                posts: posts,
+                postsCount: thread.postsCount - 1,
+              };
             });
           });
           showToast({
             variant: 'success',
-            body: 'Post got deleted successfully',
+            body: 'Reply deleted successfully',
           });
         }
       })
       .catch(() => {
-        showToast({ variant: 'error', body: 'Error while deleting post' });
+        showToast({ variant: 'error', body: 'Error while deleting reply' });
```

Comment:
Same comments as above. We should show error message.

Reply:
Can we make this error message a constant? So that if we want to update the error message, it will be a change in just one place.